page_object/tests/test_products_page.py

```diff
@@ -0,0 +1,32 @@
+from page_object.pages.products_page import Products
+
+
+def test_check_sorting_a_z(logged_in_main_page):
+    products_page = Products(logged_in_main_page.driver)
+    products_page.click_dropdown_sorting_a_z()
```

Comment:
when you encapsulate your asserts in a method, it should be clear that the method checks the result. In this row, the method called "click_dropdown_sorting_a_z" doesn't look like it checks the result. You should either move the assert from the method to the test or rename the method